### ISSUE TITLE
Initial Version

### ISSUE THREAD

**Issue by brillout** [brillout/vike#1]
I'd suggest the initial version to only be a CLI generator.

Basically: user goes to `vike.land` > user selects his stack (e.g. React + Telefunc + Prisma) > auto generation of `<span class="generated-cli">npm init vike --react --telefunc --prisma</span>`.

This repo and https://vike.land/ is already scaffolded; all commits to master will auto deploy to `vike.land`. (Note that the logos are ugly and work-in-progress.)

@mate-h What do you think? (I know you're busy; no rush here, only reply if you have time :-))

**Comment by mate-h** [comment]
Yes that's perfect since it achieves the bare minimum to be useful.
